fintech-examples/fintech-ui/src/environments/environment.ts

```diff
@@ -3,7 +3,8 @@
 // The list of file replacements can be found in `angular.json`.
 
 export const environment = {
-  production: false
+  production: false,
+  FINTECH_API: 'fintech-api-proxy'
```

Comment:
You need FINTECH_API in enviroment.prod.ts, as well. Otherwise there will be an error, if you build with prod flag.

Reply:
Got it. Fixing ...